cmd/init.go

```diff
@@ -77,22 +79,21 @@ var initCmd = &cobra.Command{
 			}
 
 			// Set the go module's name according to the repo URL
-			url, err := project.Repo.URL();if err != nil {
-
+			// TODO: better algo for this, so we can use it with ANY conn type
```

Comment:
And the better algorithm for this is `strings.TrimPrefix()`.  We can either:
* drop through a case statement switching on the prefixes we care about
* just drop through a fixed list of `strings.TrimPrefix()` statements every time.